### ISSUE TITLE
weird problem with mpd module

### ISSUE THREAD

**Issue by v0idifier** [Alexays/Waybar#664]
output gets spammed (my log file was 400mb) with
```
[warning] mpd: Out of memory
```
and
```
[warning] mpd: Connection to MPD closed
[info] mpd: Connected to MPD
```

no noticeable problem with the module usage itself.

**Comment by bbaserdem** [comment]
I also have this issue; with mpd connecting and disconnecting. I also get;

```
[error] mpd: argument not found
```

**Comment by v0idifier** [comment]
(I might have gotten the same message, I didn't look at the whole log)

**Comment by Alexays** [comment]
Should be fixed with #887
